assets/css/index.css

```diff
@@ -1,16 +1,126 @@
 :root {
   --background-color: #fff;
-  --primary-color: ;
-  --secondary-color: ;
+  --primary-color: #f15f36;
+  --secondary-color: #1fbbed;
```

Comment:
The usage of pure CSS variables is a nice addition and using the `:root` pseudoelement instead of the `html` element is better because it is more specific.